addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16541,3 +16541,13 @@ msgstr ""
 msgctxt "#38016"
 msgid "%d fps"
 msgstr ""
+
+#: system/settings/rbp.xml system/settings/imx6.xml
+msgctxt "#38120"
+msgid "Support 8 channel DTS HD audio decoding"
```

Comment:
I'm not a friend of long settings labels with probably more technical info than needed, because they tend to get even longer in translations and then scroll. Would "Enable DTS-HD audio decoding" also work? If it's 8 channel or not is of no value for mortals I think and that additional technical info is well suited in the description.